packages/webpack/src/runtime/runtime.js

```diff
@@ -201,7 +226,7 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     // The following seem harmless and are used by default: ['O', 'n', 'd', 'o', 'r', 's']
     const supportedRuntimeItems = ['O', 'n', 'd', 'o', 'r', 's']
     for (const item of supportedRuntimeItems) {
-      // @ts-expect-error - I'm not gonna do webppack's minified runtime typing
+      // @ts-ignore - I'm not gonna do webppack's minified runtime typing
```

Comment:
Why is this changed from `ts-expect-error` to `ts-ignore`? In general I see no reason to prefer the latter over the former (either we think the code should type, or we know it not to. `ts-ignore` would not let us know if the situation changes)

Reply:
I wanted to indicate that it's not that the types are missing and could show up. It's that I don't intend to have this specific thing typechecked. I don't care either way, was trying to be more specific about the intent. can undo. Let me know if you think I should, knowing the intention behind the change.

Reply:
use of `@ts-ignore` should violate [this rule](https://typescript-eslint.io/rules/ban-ts-comment/).

may be better just to wrap it in a `@type {any}`

Reply:
If that's the desired way to indicate "I explicitly don't care what type it is" - I'm fine with that.